selfie.c

```diff
@@ -5463,6 +5470,11 @@ void emit_bootstrapping() {
   // code segment starts at PK_CODE_START
   code_start = PK_CODE_START;
 
+  // store HOST_OS string literal for Selfie operating system
```

Comment:
This is too complicated. What I meant is to bootstrap in the selfie compiler all occurrences of `HOST_OS` or better `HOSTOS` to `SELFIE`. And `SELFIE` should be a global "constant" indicating that the system is hosted by selfie.

Reply:
Code like `HOSTOS == SELFIE` then becomes `SELFIE == SELFIE`.

Reply:
Got it